languages/c/reference/exercise-concepts/square-root.md

```diff
@@ -0,0 +1,17 @@
+# Concepts of square-root
+
+[Example Implementation](https://github.com/exercism/c/blob/master/exercises/square-root/src/example.c)
+
+- [Arithmetic](https://github.com/exercism/v3/blob/master/reference/concepts/arithmetic.md): Used to calculate the root
+- [Bitwise manipulation](https://github.com/exercism/v3/blob/master/reference/concepts/bitwise_manipulation.md): Bit-shifting is used several times for calculation of the output value
+- [Comparisons](https://github.com/exercism/v3/blob/master/reference/concepts/comparisons.md): Comparisons are used in while loop conditions
+- [Fixed width integer types](https://en.wikipedia.org/wiki/C_data_types#Fixed-width_integer_types): Used throughout for specific bit positioning
+- [Function Definitions](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Definitions): Used as the main entry point for the exercise
+- [Function Parameters](https://www.gnu.org/software/gnu-c-manual/gnu-c-manual.html#Function-Parameters):The definition of the function under test uses a parameter `radicand`
```

Comment:
Forgot a space after the colon.

Reply:
In the Function Parameters concept, that is.

Reply:
Well caught!